configure.ac

```diff
@@ -457,13 +457,35 @@ AC_SUBST([READLINE_LIBS])
 
 dnl Find julia
 AC_ARG_WITH([julia],
-  [AS_HELP_STRING([--with-julia@<:@=PREFIX@:>@],
-    [specify a prefix where julia can be found])],
-  [],[with_julia=no]
-)
+  [AS_HELP_STRING([--with-julia@<:@=PATH@:>@],
+    [specify a prefix for a julia binary or a julia binary])],
```

Comment:
@sebasguts do you really mean `[specify a prefix for a julia binary or a julia binary]`?

Reply:
Yes, I mean it that way.

You can either directly specify a julia binary, or its prefix, e.g., `/usr/local`

Reply:
Ah, now I got it. Parsing plain text without parentheses or commas failed. Maybe 

    specify a prefix for a julia binary, or a julia binary

or
 
    specify a julia binary or a prefix for a julia binary 

would be easier to read.

Reply:
I think I understand the problem now, sorry. I will provide a PR for that.